src/Pelagos/Entity/Person.php

```diff
@@ -765,6 +765,20 @@ public function getDataRepositories()
         return $collection;
     }
 
+    /**
+     * Get a list of the names of all Data Repositories this person is associated with.
+     *
+     * @return array
+     */
+    public function getDataRepositoryNames()
+    {
+        $dataRepositoryNames = array();
+        foreach ($this->personDataRepositories as $personDataRepository) {
+            $dataRepositoryNames[] = $personDataRepository->getDataRepository()->getName();
```

Comment:
We have only GRIIDC as the data repository, shouldn't this just be a string instead of an array

Reply:
If we add another data repository, I think we should re-architect the structure of entities. I think returning an array is confusing now.

Reply:
I'm ok with this. The twig is a little confusing, but it's fine.